src/coveralls.erl

```diff
@@ -62,6 +63,9 @@
 
 %%=============================================================================
 %% API functions
+% @doc Initialization plugin for rebar3
+-spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
+init(State) -> rebar3_coveralls:init(State).
```

Comment:
@markusn I won't work because the plugin name is coveralls and the rebar3 will start from this module